src/qt/rpcconsole.cpp

```diff
@@ -463,13 +463,15 @@ void RPCConsole::clear()
 
     // Set default style sheet
     ui->messagesWidget->document()->setDefaultStyleSheet(
+        QString::fromStdString(strprintf(
```

Comment:
Please use Qt's formatting, instead of strprintf. e.g.

    QString("....\1...." ).arg(...)